pkg/amp/bucket_set.go

```diff
@@ -93,17 +97,32 @@ func (b *bucketSet) Route(req *route.Request) route.Response {
 			return route.FAIL
 		}
 		return route.OK
+	case route.Audit:
+		if err := b.Audit(req.Flags().Get()...); err != nil {
+			msg.Error(err.Error())
+			return route.FAIL
+		}
+		return route.OK
 	case route.Info:
 		b.Info()
 		return route.OK
+	case route.Config:
+		b.Print()
+		return route.OK
+	case route.Help:
+		b.Help()
+		return route.OK
+	default:
+		msg.Error("Internal Error: Unknown command " + req.Command().String())
```

Comment:
I don't think you need the error message here, it will be confusing. I think the help is sufficient.